docs/advanced-topics/metrics.rst

```diff
@@ -184,12 +212,41 @@ The curl command looks something like:
       ]
    }'
 
-If it was submitted successfully, you should receive a ``204`` response code. 
+If it was submitted successfully, you should receive a ``204`` response code.
+
+.. _How to view metrics:
+
+Viewing Aggregated Workflow Metrics in Dockstore
+------------------------------------------------
+
+Dockstore aggregates the workflow execution metrics that platforms submit into aggregated metrics for users to view.
+
+To search for workflows with metrics, navigate to the `search <https://dockstore.org/search>`_ page and select a platform for the Execution Metrics and/or Validation Metrics facets.
+
+.. figure:: /assets/images/docs/metrics/metrics-search-facets.png
+    :alt: Metrics search facets
+
+Select a workflow and click on the Versions tab. Versions that have metrics have a check mark in the Metrics column.
+
+.. figure:: /assets/images/docs/metrics/versions-metrics-column.png
+    :alt: Metrics column in Versions table
+
+Select a version with metrics then click on the Metrics tab to view the metrics available.
+
+.. figure:: /assets/images/docs/metrics/metrics-tab.png
+    :alt: Metrics tab
+
+.. _Viewing And Updating Submitted Workflow Metrics:
+
+Viewing And Updating Submitted Workflow Metrics
+-----------------------------------------------
+
+As a platform owner who previously :ref:`submitted execution metrics to Dockstore<Getting started with submitting metrics>`, you may want to view or update submitted executions.
 
 .. _View submitted execution:
 
-How do I view a submitted execution?
-------------------------------------
+Viewing a submitted execution
```

Comment:
Took out "I" here but  added it earlier with "Why Would I, a Platform Owner, Want to Submit Workflow Metrics?"
Just curious

Reply:
The diff is misleading, "Why Would I, a Platform Owner, Want to Submit Workflow Metrics?" wasn't changed (it always had "I")

It doesn't match with the other titles though, so I will remove "I"